make_prg/subcommands/from_msa.py

```diff
@@ -67,12 +70,28 @@ def register_parser(subparsers):
         action="store",
         help="Prg file name. Default: MSA file name",
     )
+    subparser_msa.add_argument(
+        "-S",
+        "--seqid",
+        help="Sequence identifier to use for the output sequence/PRG. Default is the file name",
+    )
     subparser_msa.add_argument(
         "--no_overwrite",
         dest="no_overwrite",
         action="store_true",
-        help="By default, a prg file which already exists is overwritten and the prg is recomputed. Use this option to keep an existing prg file instead.",
+        help="Do not replace an existing prg file",
+    )
+    subparser_msa.add_argument(
+        "--summary", help="Write a summary file", action="store_true"
```

Comment:
Do you guys ever use summary file? We could remove it entirely potentially, or just write it out to logging, as its two lines. @leoisl

Reply:
I have never used it. Hence why I changed the default behaviour. 

If Leandro never uses it either I vote just get rid of it. It's in the commit history if we ever want it back

Reply:
I've never used it, I vote to remove it also